checker/tests/nullness/Issue2215.java

```diff
@@ -0,0 +1,34 @@
+// Test Case for Issue2215
+// https://github.com/typetools/checker-framework/issues/2215
+// @skip-test
+
+import org.checkerframework.checker.nullness.qual.MonotonicNonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class ParentClass {
+
+    @MonotonicNonNull Object parentClassField;
+
+    ParentClass(@Nullable Object o) {
+        this.parentClassField = o; // should be allowed
+    }
+}
+
+public class Issue2215 extends ParentClass {
+
+    @MonotonicNonNull Object f;
+    @MonotonicNonNull Object g = null;
+    @MonotonicNonNull Object h = new Object();
+
+    Issue2215(@Nullable Object o) {
+
+        super(o); // should be allowed
+        this.parentClassField = o; // should be allowed
+        this.f = o; // should be allowed
+        this.f = null; // should be allowed
+        this.g = o; // should be allowed
+        this.g = null; // should be allowed
+        this.h = o; // should not be allowed
```

Comment:
This is not how an error is indicated in a Checker Framework test case.
See any existing test case, or see https://raw.githubusercontent.com/typetools/checker-framework/master/checker/tests/README .